heat/core/dndarray.py

```diff
@@ -1351,7 +1450,9 @@ def __getitem__(self, key):
                 else:
                     new_split = self.split
 
-                if key in range(chunk_start, chunk_end) and self.split == 0:  # only need to adjust the key if split==0
+                if (
```

Comment:
This also seems wrong, comment should be above

Reply:
While the comments should certainly not be put in the way that Black did automatically, one can argue in the case of branch statements that one should put the explanation of the condition *inside* the corresponding branch body.